rotmat.py

```diff
@@ -362,9 +365,9 @@ def test_euler():
     '''check that from_euler() and to_euler() are consistent'''
     m = Matrix3()
     from math import radians, degrees
-    for r in range(-179, 179, 3):
-        for p in range(-89, 89, 3):
-            for y in range(-179, 179, 3):
+    for r in list(range(-179, 179, 3)):
+        for p in list(range(-89, 89, 3)):
+            for y in list(range(-179, 179, 3)):
```

Comment:
And again...

Py2 range: `list(xrange())`, In Py 3 range() works like xrange().
I do not see any reason to make list from range object.